src/features/organizations/components/OrganizationsList.tsx

```diff
@@ -21,7 +21,7 @@ const OrganizationsList = ({ model }: UserOrganizationsProps) => {
           <Box style={{ margin: '30px' }}>
             <Typography variant="h3">{messages.page.title()}</Typography>
             <List>
-              {data.map((org: ZetkinOrganization) => {
+              {data.map((org: ZetkinMembership['organization']) => {
```

Comment:
This isn't beautiful, but this entire component will be going away anyway (we won't have the /organize page now that we have the switcher in the menu) so I only solved it in the quickest way possible.